lib/routers/profile/pil.js

```diff
@@ -151,7 +120,7 @@ router.get('/:pilId',
   permissions('pil.read'),
   attachEstablishmentDetails,
   (req, res, next) => {
-    res.response = attachReviewDue(req.pil);
+    res.response = attachReviewDue(2)(req.pil);
```

Comment:
This one should be `3`, no?
```suggestion
    res.response = attachReviewDue(3)(req.pil);
```

Reply:
I was going with your suggestion above

Reply:
"Otherwise the count on the dashboard and the number of due indicators in the list won't align with one another"

Reply:
This is for the single PIL view though isn't it? Not the list.

Hence this one still needs to be 3 months so that the PIL holder gets the extra month of lead time. Unless _that_ 3 months is somewhere else and I'm misunderstanding the code.

Reply:
nope - you're right. I was thinking it was for the 2 counts, not a single model